scripts/alert_emails/utils.ts

```diff
@@ -90,8 +90,10 @@ function generateAlertEmailContent(
     last_updated: lastUpdated,
     location_url: `${locationURL}?utm_source=risk_alerts&utm_medium=email`,
     subscribe_link: `https://covidactnow.org/alert_signup?utm_campaign=risk_alert_forward_subscribe&utm_source=risk_alerts&utm_medium=email`,
-    unsubscribe_link: `${unsubscribeURL}?email=${encodeURI(emailAddress)}`, // would be nice to know dev/staging/prod
-    feedback_subject_line: encodeURI(
+    unsubscribe_link: `${unsubscribeURL}?email=${encodeURIComponent(
+      emailAddress,
+    )}`, // would be nice to know dev/staging/prod
+    feedback_subject_line: encodeURIComponent(
```

Comment:
We were using `encodeURI()` in a few places where we should have been using `encodeURIComponent()`... and so e.g. unsubscribe links in emails sent to people with `+` in their email didn't work.

Reply:
oof, oh no... good catch!